### PR TITLE
Repurposed GetMessagesAsync()

### PR DESCRIPTION
**Pull request by jcryer**
# Summary
Allows bots to access messages from the end of a channel without using Channel.LastMessageId.

# Details
LastMessageId was dodgy and SSG forgot to add this.

### CODE REVIEW

File: DSharpPlus/Entities/DiscordChannel.cs [hunk 1]
```diff
@@ -316,9 +316,12 @@ public Task<IReadOnlyList<DiscordMessage>> GetMessagesAfterAsync(ulong after, in
...
-        public Task<IReadOnlyList<DiscordMessage>> GetMessagesAsync(int limit = 100, ulong? before = null, ulong? after = null, ulong? around = null) =>
-            _getMessagesAsync(limit, before, after, around);
+        /// <summary>  
+        /// Returns a list of messages from the last message in the channel.
+        /// <param name="limit">The amount of messages to fetch, up to a maximum of 100</param>
+        /// </summary> 
+        public Task<IReadOnlyList<DiscordMessage>> GetMessagesAsync(int limit = 100) =>
+            _getMessagesAsync(limit, null, null, null);
 
         private Task<IReadOnlyList<DiscordMessage>> _getMessagesAsync(int limit = 100, ulong? before = null, ulong? after = null, ulong? around = null)
```
Comment (Emzi0767) [inline DSharpPlus/Entities/DiscordChannel.cs]: Private methods should still follow PascalCase naming convention, please do that.

**Comment by NaamloosDT** [comment]
other than naming, lgtm

**Comment by NaamloosDT** [comment]
Might wanna fix that build ;)

**Comment by jcryer** [comment]
Yeah, it was because I made my last edit while you guys had borked the lib. Fixed with a merge into my branch!

**Pull request merged**